steps/vtr-build.sh

```diff
@@ -13,3 +13,15 @@ if [ $FAILURE -ne 0 ]; then
         make CMAKE_PARAMS="-DVTR_ASSERT_LEVEL=3" -j1
         exit 1
 fi
+
+echo "----------------------------------------"
```

Comment:
Move this to the beginning of `vtr-build.sh`?

Reply:
Done